config/application.rb

```diff
@@ -136,6 +137,12 @@ class Application < Rails::Application
     # Add apidoc hash in headers for smarter caching
     config.middleware.use "Apipie::Middleware::ChecksumInHeaders"
 
+    Foreman::Logging.configure(:log_directory => "#{Rails.root}/log")
+    config.colorize_logging = Foreman::Logging.config[:colorize]
+
+    config.logger = ::Logging.logger['app']
+    config.active_record.logger = ::Logging.logger['sql']
```

Comment:
There is a rails-logging gem that does this sort of (https://github.com/TwP/logging-rails#usage) on a per file basis but I thought it might be overkill.